### DISCUSSION TITLE
How is this bundling up?

### DISCUSSION THREAD

**Discussion by biratdatta** [xyflow/xyflow#4597] [General]
I am curious to know how you are bundling up (Vite I checked the package.json) and making it an NPM package. I am facing multiple issues with the import of reactflow inside the flowise. It Would help a lot if someone can answer that.

**Comment by bcakmakoglu** [comment]
You'll have to describe your issue a bit more in detail. At least I don't really understand what the question here is or what problems you are facing.

**Reply by biratdatta** [reply]
Hey I wanted to know what kind of bundler is being used here to bundle it so that I can bundle up and make it into an NPM package and use it for custom usage @bcakmakoglu

**Reply by bcakmakoglu** [reply]
The library uses [Rollup](https://rollupjs.org/) to create a distribution, check the `build` command in, for example, `packages/react/package.json`,

```json
 "build": "rollup --config node:@xyflow/rollup-config --environment NODE_ENV:production && npm run css",
```

**Reply by biratdatta** [reply]
thanks will look into and try it out!
